test/kata3.numberToReversedDigits.test.js

```diff
@@ -1,5 +1,15 @@
 const { numberToReversedDigits } = require("../src");
 
 describe("numberToReversedDigits", () => {
-  test("returns a reversed array of the number's digits", () => {});
-});
\ No newline at end of file
+  it("returns a reversed array of the numbers digits", () => {
+    expect(numberToReversedDigits(123)).toStrictEqual([3, 2, 1]);
+  });
+  it("returns 1-9 in reverse as an array", () => {
+    expect(numberToReversedDigits(123456789)).toStrictEqual([
+      9, 8, 7, 6, 5, 4, 3, 2, 1,
+    ]);
+  });
+  it("returns odd numbers in reverse as an array", () => {
```

Comment:
You should always add a clear line between each of your test blocks to make it easier to read:
```
  });
  
  it("returns 1-9 in reverse as an array", () => {
    expect(numberToReversedDigits(123456789)).toStrictEqual([
      9, 8, 7, 6, 5, 4, 3, 2, 1,
    ]);
  });
  
  it("returns odd numbers in reverse as an array", () => {
```